docs/resources/user.md

```diff
@@ -22,33 +25,34 @@ resource "datadog_user" "foo" {
 }
 ```
 
-## Argument Reference
+## Schema
 
-The following arguments are supported:
+### Required
 
--   `email`: (Required) Email address for user
--   `name`: (Optional) Name for user
--   `roles`: (Optional) A list a role IDs to assign to the user.
--   `disabled`: (Optional) Whether the user is disabled
--   `send_user_invitation`: (Optional) Whether an invitation email should be sent when the user is created. Defaults to `true`.
--   `handle`: (Deprecated) The user handle, must be a valid email.
--   `is_admin`: (Deprecated) (Optional) Whether the user is an administrator. **Warning**: the corresponding query parameter is ignored by the Datadog API, thus the argument would always trigger an execution plan.
--   `role`: (Deprecated) Role description for user. **Warning**: the corresponding query parameter is ignored by the Datadog API, thus the argument would always trigger an execution plan.
--   `access_role`: (Deprecated) Role description for user. Can be `st` (standard user), `adm` (admin user) or `ro` (read-only user). Default is `st`. `access_role` is ignored for new users created with this resource. New users have to use the `roles` attribute.
+- **email** (String) Email address for user.
 
-## Attributes Reference
+### Optional
 
-The following attributes are exported:
+- **access_role** (String) Role description for user. Can be `st` (standard user), `adm` (admin user) or `ro` (read-only user). Default is `st`. `access_role` is ignored for new users created with this resource. New users have to use the `roles` attribute.
```

Comment:
should we denote that this field is deprecated?

Reply:
Good note, this PR was autogenerated based on whats defined in the Schema. I just opened a PR to add Deprecated to this field and I can regenerate this once thats merged - https://github.com/DataDog/terraform-provider-datadog/pull/834